bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/dto/PrmDetail.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.linky.internal.dto;
+
+import java.util.ArrayList;
+
+/**
+ * The {@link PrmDetail} holds detailed informations about prm configuration
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+public class PrmDetail {
+    public record Adresse(String ligne2, String ligne3, String ligne4, String ligne5, String ligne6) {
```

Comment:
There is no ligne1 ?

Reply:
...surprizingly...no. I imagine ligne1 would be the customer name

Reply:
Minor, maybe change to `line` over `ligne`.

Reply:
Done

Reply:
This is the name used by API in its response. A French service.

Reply:
Seems reasonable and maybe mandatory, better to revert.

Reply:
Done